buildspecs/linux_ppc-64.spec

```diff
@@ -58,7 +58,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<property name="graph_req.aux0" value=""/>
 		<property name="graph_req.aux1" value=""/>
 		<property name="graph_req.build" value="build:java8sr5fp27+"/>
-		<property name="graph_req.build2" value="build:java8sr5fp27+"/>
+		<property name="graph_req.build2" value="build:java8sr6fp5+"/>
```

Comment:
@vsebe this capability should be sr7 and not sr6. Also, can you please change `build` as well to be consistent, even if it's not used atm.

Reply:
You can squash my change into a single commit.

Reply:
done